test/frontend.js

```diff
@@ -94,44 +84,36 @@ describe('Frontend', function () {
   })
   n.on('page', (type, msg, stack) => {
     if (type === 'error') {
-      console.log('!! [NIGHTMARE] page error:', msg, 'stack:', stack)
+      console.error('\x1b[31m', '!! [NIGHTMARE] page error:', msg, 'stack:', stack, '\x1b[0m')
     }
   })
   n.on('console', (type, args) => {
     if (type === 'error') {
       var idx = -1
       if (args.indexOf) idx = args.indexOf('[NIGHTMARE NOTE]: ')
       if (idx !== -1) {
-        console.log('!! ' + args.slice(idx))
+        console.warn('\x1b[34m', '!! ' + args.slice(idx), '\x1b[0m')
       } else {
-        console.log('!! [NIGHTMARE] console error:', args)
+        console.error('\x1b[31m', '!! [NIGHTMARE] console error:', args, '\x1b[0m')
       }
     }
   })
 
   after(() => { n.end() })
 
-  it('Should start the backend server if necessary', function () {
+  before(() => {
     return require('../backend/index.js')
   })
-  let username = `User`
-
-  describe.skip('New user page', function () {
-    it('Should create user George', function () {
-      this.timeout(5000)
-      return n.use(signup('George', 'george@lasvegas.com', '$$111$$'))
-    })
 
-    it.skip('Should fail to create George again', function () {
-      return n.click('.signup button.submit')
-        .wait(() => document.getElementById('serverMsg').className.indexOf('danger') !== -1)
-    })
-  })
+  let username = `User`
 
   describe('Sign up Test', function () {
-    it('Should register User', function () {
+    it('Should register User', async function () {
       this.timeout(10000)
-      return n.use(signup(username, 'test@testgroupincome.com', 'testtest'))
+      const signedup = await n
+        .use(signup(username, 'test@testgroupincome.com', 'testtest'))
+        .evaluate((el) => !!document.querySelector(el), elT('homeLogo'))
+      should(signedup).equal(true)
```

Comment:
Same, I see no reason to modify this. The `signup` should already have a wait for the condition of being signed in. If it doesn't, then the `signup` functions needs to be modified, not this part. Please undo